src/tools/rbd/action/Group.cc

```diff
@@ -254,7 +254,7 @@ int execute_remove_image(const po::variables_map &vm,
       vm, at::ARGUMENT_MODIFIER_NONE, arg_index);
 
   if (!image_id.empty() && has_image_spec) {
-    std::cerr << "rbd: trying to access image using both name and id. "
+    std::cerr << "rbd: trying to access image using either name or id. "
```

Comment:
... actually, now that I see the condition in the ```if``` clause, the original wording was correct. It's an error to try to access the image by **both** the image id and name.

Reply:
Well, it sounds more reasonable, I think it is just a error/warning message for user. Repushed, thanks.